.travis.yml

```
@@ -7,7 +7,8 @@ sudo: false
 cache: pip
 
 before_install:
-- pip install --upgrade flit
+- &install-flit
```

Can we have an explicit block literal here?
```suggestion
- &install-flit >-
```